heeji/2차원으로-만들기.js

```diff
@@ -0,0 +1,7 @@
+function solution (num_list, n) {
+    let newArray = [];
+    for (i = 0; i < num_list.length; i + n) {
+        newArray.push(num_list.splice(i, n));
+    }
+    return newArray;
+}
```

Comment:
- 저랑 똑같은 방법으로 푸셨네요 👍 잘푸셨습니다
- for문 이용 시 `let` 키워드 꼭 넣어주세요!! 넣지 않으면 i는 블록레벨스코프가 아니게 되어 밖에서 for문 밖에서 i 호출 시 존재하게 되어버립니다!

```suggestion
function solution (num_list, n) {
    let newArray = [];
    for (let i = 0; i < num_list.length; i + n) {
        newArray.push(num_list.splice(i, n));
    }
    return newArray;
}
```